package.json

```diff
@@ -54,10 +54,11 @@
     "license-check": "jsgl --local ."
   },
   "dependencies": {
-    "@google-cloud/common": "^0.21.0",
+    "@google-cloud/common": "^0.23.0",
     "console-log-level": "^1.4.0",
     "is": "^3.2.1",
-    "lodash.has": "^4.5.2"
+    "lodash.has": "^4.5.2",
+    "request": "^2.88.0"
```

Comment:
@fhinkel could we start actually using teeny-request here?

Reply:
Soon!